ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-workflow.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/user-labels-tags/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-workflow.yaml",
```

Comment:
Suggest to create a dedicated folder for `user-labels-tags/filestore-csi` or `filestore-csi/user-labels-tags` to save the install workflow.

Reply:
The workflow is for user defined labels & tags testing, nothing about filestore-csi. Could you please clarify?

Reply:
The [workflow](https://github.com/openshift/release/pull/42819/files#diff-a53d8367e2f8a5c3436a55ddde4ab4c86c6ea9797fd30ccd08c4d330a198d098R6) appended `storage-conf-csi-optional-gcp-filestore` chain, the provision chain is nothing about `filestore-csi`, right?